torch/_inductor/overrides.py

```diff
@@ -897,6 +931,27 @@ def fuse_binary_inplace(gm: torch.fx.GraphModule):
     return gm
 
 
+def packed_module(gm: torch.fx.GraphModule):
```

Comment:
How about providing a dedicated transformation function for packed linear first? We can consider to generalize it later when more cases are applicable?

Reply:
I add a packed Conv at nest PR, so I think it is ok.